www.foia.gov/Makefile

```diff
@@ -0,0 +1,5 @@
+clean:
+	perl -pi -e 's,https://www.foia.gov/,,g' *html
```

Comment:
The existing site seems to have all the top-level .html files referring to other files that are all top-level. Putting the leading slash on `/foia.css` meant that if you served the DocumentRoot of the site at any path other than the root of the domain, the relative links would no longer be relative.

The `make clean` target is irrelevant at this point anyway; I can remove it. It was there only while I was doing the initial migration.